app/controllers/document_images_controller.rb

```diff
@@ -22,8 +22,7 @@ def create
   end
 
   def update
-    document = Document.find_by_param(params[:document_id])
-    image = Image.find_by!(id: params[:id], document_id: document.id)
+    image = Image.find(params[:id])
```

Comment:
This originally had the document lookup to ensure we had an image for that document - since we have both document_id and image id in the url. 

To maintain this it might be best that this does `image = Document.find_by_param(params[:document_id]).images.find(params[:id])`

what do you think?